source/reference/command/cleanupOrphaned.txt

```diff
@@ -298,4 +298,19 @@ the following:
      nextKey = result.stoppedAtKey;
    }
 
+
+Control performance impact
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to tune the performance impact of :dbcommand:`cleanupOrphaned`
+with two server parameters `rangeDeleterBatchSize</reference/parameters/#param.rangeDeleterBatchSize>`
```

Comment:
You can actually just go :parameter:\`rangeDeleterBatchSize\` and  :parameter:\`rangeDeleterBatchDelayMS\`

As @devkev  mentioned in the DOCS-12492, since these settings would also affect chunk migration, we probably should mention this fact -- in case people really just want to set them temporarily for the command - but want whatever for the chunk migration.

Also, probably the content should go with the Performance section -- 
https://github.com/mongodb/docs/pull/4032/files#diff-01c6bc4c93806419712b7070b0efe7e5L113

Reply:
Oh -- and does git pull --rebase not work?  We tend to keep our commits 1 per fix.